src/empirical.jl

```diff
@@ -30,7 +82,7 @@ struct EmpiricalVariogram
   ordinate::Vector{Float64}
   counts::Vector{Int}
 
-  function EmpiricalVariogram(sdata, var₁, var₂, nlags, maxlag, distance)
+  function EmpiricalVariogram(sdata, var₁, var₂, nlags, maxlag, distance, nearestsearch)
```

Comment:
I think we should make this a keyword argument as opposed to a positional argument.

Reply:
I did that here, where all the other keywords are set as well:

```julia
EmpiricalVariogram(sdata::AbstractData, var₁::Symbol, var₂::Symbol=var₁;
                   nlags=20, maxlag=nothing, distance=Euclidean(), nearestsearch=nothing) =
  EmpiricalVariogram(sdata, var₁, var₂, nlags, maxlag, distance, nearestsearch)
```

Not even sure if you can use keywords in a inner constructor actually.